### PR TITLE
feat: Adds support for AtomOne Gov SDK module

### PR DESCRIPTION
**Pull request by clockworkgr**
This PR adds support for viewing and interacting with proposals on the AtomOne chain by implementing support for the AtomOne version of the Cosmos SDK gov module

Followed the same principles/flow as https://github.com/hicommonwealth/commonwealth/pull/9097

Assumes the AtomOne community has Cosmos Governance version `v1atomone`

### CODE REVIEW

File: libs/model/src/services/commonProtocol/contestHelper.ts [hunk 1]
```diff
@@ -347,15 +347,22 @@ export const rollOverContest = async (
       ? contestInstance.methods.endContest()
       : contestInstance.methods.newContest();
 
-    let gasResult;
+    let gasResult = BigInt(300000);
```
Comment (Rotorsoft) [inline libs/model/src/services/commonProtocol/contestHelper.ts:350]: should we move magic constant to config var?

File: libs/model/src/services/commonProtocol/contestHelper.ts [hunk 2]
```diff
@@ -347,15 +347,22 @@ export const rollOverContest = async (
...
       });
     } catch {
-      return false;
+      //eslint-disable-next-line
+      //@ts-ignore no-empty
     }
+
     const maxFeePerGasEst = await estimateGas(web3);
+
+    if (gasResult < BigInt(100000)) {
```
Comment (Rotorsoft) [inline libs/model/src/services/commonProtocol/contestHelper.ts:362]: same here

File: libs/model/src/thread/GetThreads.query.ts [hunk 3]
```diff
@@ -184,7 +185,12 @@ export function GetThreads(): Query<typeof schemas.GetThreads> {
                 'thread_id', TT.id,
                 'content_id', CA.content_id,
                 'start_time', CON.start_time,
-                'end_time', CON.end_time
+                'end_time', CON.end_time,
```
Comment (Rotorsoft) [inline libs/model/src/thread/GetThreads.query.ts:188]: this is already merged in master

**Review by Rotorsoft — APPROVED** [review]
LGTM
... wondering if at some point we would consider modularizing the client architecture to support these types of plugins or micro frontends, allowing us to dynamically import components such as AtomOne views/widgets/flows on demand.

**Comment by clockworkgr** [comment]
closing in favour of https://github.com/hicommonwealth/commonwealth/pull/10180

**Pull request closed without merging**